packages/select/src/FilterMultiSelect/components/ListBox/ListBox.tsx

```diff
@@ -1,18 +1,34 @@
 import React, { useState, useEffect } from "react"
-import { Node } from "@react-types/shared"
+import { Collection } from "@react-types/shared"
 import classNames from "classnames"
-import { ItemType } from "../../../types"
+import { MultiSelectItem } from "../../../types"
 import { useSelectionContext } from "../../provider/SelectionProvider"
 import styles from "./ListBox.module.scss"
 
+type ListBoxItems = {
+  selectedItems: MultiSelectItem[]
+  unselectedItems: MultiSelectItem[]
+  disabledItems: MultiSelectItem[]
+  allItems: MultiSelectItem[]
+  hasNoItems: boolean
+}
+
 export interface ListBoxProps {
-  children: (items: {
-    selectedItems: Array<Node<ItemType>>
-    unselectedItems: Array<Node<ItemType>>
-    disabledItems: Array<Node<ItemType>>
-    allItems: Array<Node<ItemType>>
-    hasNoItems: boolean
-  }) => React.ReactNode
+  children: (items: ListBoxItems) => React.ReactNode
+}
+
+/** get items from keys and filter out nullish or undefinded values
+ ** this is required now `collection.getItem()` has a nullish return value
+ */
```

Comment:
No need for this comment :) It's enough to explain it in the PR desc, but not needed in the codebase since the function is pretty self-explanatory.